src/common/api/queryParamsBuilder.ts

```diff
@@ -29,7 +30,15 @@ export class QueryParamsBuilder {
 
   public setFilterParam(filters: Filter[]): QueryParamsBuilder {
     filters.forEach(f => {
-      this.setParam(`${f.attr}__${f.op}`, f.value);
+      if (f.attr === 'phoneNumber') {
```

Comment:
This is mixing concerns here.

queryParamBuilder should not need to have an attribute named a special name to do the filtering you desire. Instead create a different `FilterUI` component instead of `StringFilter` that does this transformation for you.

Reply:
Hmm. I bumped into this problem initially. If I create a new filter and do the transform there, the result will be shown to the user via the text input that's a part of the FilterDropdown. That isn't what I want since the user expects their input to match the phone number format that's shown in the UI, ex. (559) 394-3984. I need the transform to happen when the query to the backend is being generated. For reference, I need it to be transformed to the following format: +15593943984. It doesn't look like the FilterUI methods are able to support this situation. Do you see what I mean? Or, am I missing something?

Reply:
I think we should just remove the phone filter aspect of this. I don't see this doing anything that the search doesn't do. And for phone numbers you usually are going to want a "contains" filter anyway.